app/api/srch/search.rb

```diff
@@ -244,7 +244,7 @@ class Search < Grape::API
               score: model.answers.length,
               latitude: model.lat,
               longitude: model.lon,
-              blurred: model.blurred?
+              blurred: false
```

Comment:
ah, here, we do want to pass this `blurred?` value along because eventually we'll be displaying them differently using https://github.com/publiclab/leaflet-environmental-layers/ -- make sense? Thank you!

Reply:
yes, make sense! To be fair, I don't even think this is the place for this. The `user` model already has an instance class to know that here:

https://github.com/publiclab/plots2/blob/1d5eeeb148cb87be251800b4c047f7c519fd15b3/app/models/user.rb#L204

So whenever it's necessary to know that, it's possible to just call that `user.blurred?` Can I remove it from this file? What do you think?

Reply:
Ah, but so we actually will need to be able to know it's blurred status from a JavaScript call, not a Ruby call, which is why this is here. Make sense? Probably specifically the call I changed in https://github.com/publiclab/plots2/pull/3940

So once we preserve this we can merge it!

Reply:
oh, yeah, I remember now why I removed this line.

https://github.com/publiclab/plots2/blob/master/app/services/search_service.rb#L109

You see, the way the method is defined, it only returns the nodes from users that don't have the blurred tag set. That was something that was already there, I just organized the code. That's why it was set to false before...

Reply:
I don't know what's happening and I'm a little busy right now. Maybe I should close this since you already added the changes to the JS call?

Reply:
No problem at all, I think i can just make a small adjustment and get this passing -- actually i realized this sub thread of comments is referring to an earlier state, so that line deletion is not even present! Picking up below and I'll go ahead and get this resolved. Thanks for checking in, and apologies for bothering while you're busy, i just didn't want to neglect this one. 👍

Reply:
No need for you to apologize, sorry I couldn't finish this, but I have a lot going on right now and I can't separate a time to put everything together... hope it won't be too much problem. Cheers!

Reply:
Good luck with everything!!! fingers 🤞 and i think this commit will pass tests 👍

Reply:
awesome, thank you!